packages/design/src/DataTable/Pager/Pager.tsx

```diff
@@ -22,10 +22,15 @@ export function Pager({
 }: State) {
   const isFetchingEnabled = onFetchMore && fetchStatus !== 'disabled';
   return (
-    <Flex
-      justifyContent={isFetchingEnabled ? 'space-between' : 'end'}
-      width="100%"
-    >
+    <Flex alignItems="center">
+      <Text
+        typography="body2"
+        color="primary.contrastText"
+        mr={isFetchingEnabled ? 1 : 2}
```

Comment:
```suggestion
        mr={isFetchingEnabled ? 1 : 3}
```
so the fetch button and without fetch button width doesn't jump too much